modules/apps/27-interchain-accounts/controller/keeper/msg_server.go

```diff
@@ -25,15 +25,15 @@ func NewMsgServerImpl(keeper *Keeper) types.MsgServer {
 }
 
 // RegisterAccount defines a rpc handler for MsgRegisterAccount
-func (k msgServer) RegisterAccount(goCtx context.Context, msg *types.MsgRegisterAccount) (*types.MsgRegisterAccountResponse, error) {
+func (s msgServer) RegisterAccount(goCtx context.Context, msg *types.MsgRegisterAccount) (*types.MsgRegisterAccountResponse, error) {
```

Comment:
Ah, nice I see that you're renaming it here now... So forget about my comment [here](https://github.com/cosmos/ibc-go/pull/2251#discussion_r966891839).